src/rt/sections_elf_shared.d

```diff
@@ -36,6 +40,15 @@ else version (FreeBSD)
     import core.sys.freebsd.sys.elf;
     import core.sys.freebsd.sys.link_elf;
 }
+else version (OSX)
```

Comment:
Not sure I understand when you're using `SharedDarwin` and when you're using `OSX`.

Reply:
Yeah I noticed the inconsistency too. It's not my code, and as you know, I have no interest in & knowledge of the different Mac OS flavours (desktop, iOS, watchOS, whatever).

Reply:
It is your code, you added `SharedDarwin`.